app/models/qdm/basetypes/data_element.rb

```diff
@@ -29,6 +29,12 @@ def codes
       dataElementCodes.collect { |code| QDM::Code.demongoize(code) }
     end
 
+    # Helper method that returns the first code on this data element as a QDM::Code
+    # object.
+    def code
```

Comment:
`code` here seems to relate to `code` in the js, but `codes` here *does not* seem to relate to `codes` in the js (this seems to ensure it's a QDM::Code for both of these, but `codes` in the js performs no transforms). Maybe I'm missing something w front end vs back end structure, though.

Reply:
Agreed. This should be removed, as it is unnecessary.

Reply:
.code is used in the cqm-execution by the results_helpers. It might be possible to use getCode instead there and only grab the first code returned

Reply:
This is the backend one, this can be removed.